patchelf/plan.sh

```diff
@@ -1,10 +1,12 @@
 pkg_name=patchelf
+pkg_description="A small utility to modify the dynamic linker and RPATH of ELF executables"
 pkg_origin=core
-pkg_version=0.8
+pkg_version=0.9
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('gplv3')
```

Comment:
While you are here you might want to change this to the SPDX identifiers we've been trying to use.